cypress/integration/plugins/observability-dashboards/6_notebooks.spec.js

```diff
@@ -34,7 +34,7 @@ const makeTestNotebook = () => {
 
   cy.contains(`Notebook "${notebookName}" successfully created`);
 
-  cy.get('h1[data-test-subj="notebookTitle"]')
+  cy.get('h3[data-test-subj="notebookTitle"]')
```

Comment:
```suggestion
  cy.get('[data-test-subj="notebookTitle"]')
```
Why do you need the html tag when the test subject exists?

Reply:
removed